### PR TITLE
Add GraphQL support documentation

### PR DESCRIPTION
**Pull request by shucon**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Documentation for: https://github.com/SeleniumHQ/selenium/issues/8020

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/grid/grid_4/graphql_support.en.md
Comment (diemol) [inline docs_source_files/content/grid/grid_4/graphql_support.en.md]: I'd prefer to merge this when we have examples, leaving docs in temporal states is not nice :)

Comment (shucon) [reply docs_source_files/content/grid/grid_4/graphql_support.en.md]: Done.

File: docs_source_files/content/grid/grid_4/graphql_support.en.md
Comment (shs96c) [inline docs_source_files/content/grid/grid_4/graphql_support.en.md:50]: The query should be properly formatted JSON:

`curl --data '{"query": "{ grid { totalSlots, usedSlots } }"}'`

would be the equivalent

**Review by shs96c — CHANGES_REQUESTED** [review]

File: docs_source_files/content/grid/grid_4/graphql_support.en.md
Comment (shucon) [reply docs_source_files/content/grid/grid_4/graphql_support.en.md:50]: Okay, I'll make the changes once the other PRs are finalized.

**Comment by AutomatedTester** [comment]
@shucon once your updates are made I will merge this PR. It doesnt need the other code ones  to be landed as they are not going to have any substantive changes(if any)

**Comment by AutomatedTester** [comment]
landed in f8254fbff48e2c349d9383fc59bd9d3944c108be

**Pull request closed without merging**
